CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -373,96 +373,96 @@
         <note>Persnickety Preferences header</note>
       </trans-unit>
       <trans-unit id="Crew" xml:space="preserve">
-        <source>Crew</source>
+        <source>Crew</source><target>Załoga</target>
         <note/>
       </trans-unit>
       <trans-unit id="Custom icons" xml:space="preserve">
-        <source>Custom icons</source>
+        <source>Custom icons</source><target>Spersonalizowane ikony</target>
         <note>Shown on the pre-sales screen to get users to subscribe</note>
       </trans-unit>
       <trans-unit id="Data provided by The Movie Database" xml:space="preserve">
-        <source>Data provided by The Movie Database</source>
+        <source>Data provided by The Movie Database</source><target>Dane dostarczone przez The Movie Database</target>
         <note/>
       </trans-unit>
       <trans-unit id="Data provided by:" xml:space="preserve">
-        <source>Data provided by:</source>
+        <source>Data provided by:</source><target>Dane dostarczone przez:</target>
         <note/>
       </trans-unit>
       <trans-unit id="Default Spoiler Settings" xml:space="preserve">
-        <source>Default Spoiler Settings</source>
+        <source>Default Spoiler Settings</source><target>Domyślne ustawienia spoilerów</target>
         <note/>
       </trans-unit>
       <trans-unit id="Delete recent searches" xml:space="preserve">
-        <source>Delete recent searches</source>
+        <source>Delete recent searches</source><target>Usuń ostatnie wyszukiwania</target>
         <note/>
       </trans-unit>
       <trans-unit id="Delete search history" xml:space="preserve">
-        <source>Delete search history</source>
+        <source>Delete search history</source><target>Usuń historię wyszukiwania</target>
         <note/>
       </trans-unit>
       <trans-unit id="Details" xml:space="preserve">
-        <source>Details</source>
+        <source>Details</source><target>Szczegóły</target>
         <note/>
       </trans-unit>
       <trans-unit id="Discover" xml:space="preserve">
-        <source>Discover</source>
+        <source>Discover</source><target>Odkryj</target>
         <note/>
       </trans-unit>
       <trans-unit id="Done" xml:space="preserve">
-        <source>Done</source>
+        <source>Done</source><target>Zrobione</target>
         <note/>
       </trans-unit>
       <trans-unit id="Each TV show can have their own settings, set by tapping the `Hide` `Spoilers…` button when looking at a show or episode." xml:space="preserve">
-        <source>Each TV show can have their own settings, set by tapping the `Hide` `Spoilers…` button when looking at a show or episode.</source>
+        <source>Each TV show can have their own settings, set by tapping the `Hide` `Spoilers…` button when looking at a show or episode.</source><target>Każdy serial może mieć swoje własne ustawienia, dostosowywane przez stuknięcie przycisku ‚Ukryj’ ‚Spoilery’ podczas patrzenia na serial lub jego odcinek.</target>
```

Comment:
Tutaj te ` to są "backticks". Jak chcesz używać "quotes", to spacja nie musi być wykluczona: ‚Ukryj spoilery’ wtedy.